styles/themes/teammember/_login-button.scss

```diff
@@ -0,0 +1,36 @@
+$large-button-font-size: 21px;
```

Comment:
Do we need all this? If we moved most of large button styling to style-guide then isn't most of this redundant?